src/wazuh_modules/vulnerability_detector/wm_vuln_detector_nvd.c

```diff
@@ -2120,9 +2125,24 @@ bool wm_vuldet_check_generic_package(char * agent_id, sqlite3_stmt *pStmt, sqlit
                     newPkg->nvd_cond->start_operation = (start_including) ? START_INCLUDED : START_EXCLUDED;
                 }
 
-                if (wm_vuldet_add_cve_node(newPkg, cve, cve_table) == -1) {
-                    merror(VU_INSERT_PACKAGE_ERROR, agent_id, pkg_reference, cve, "NVD");
+                int result = wm_vuldet_add_cve_node(newPkg, cve, cve_table);
+
+                switch (result) {
+                case -1:
+                    mterror(WM_VULNDETECTOR_LOGTAG,VU_INSERT_PACKAGE_ERROR, agent_id, pkg_reference, cve, "NVD");
+                    wm_vuldet_free_cve_node(newPkg);
+                    break;
+                case 1:
+                    mtdebug1(WM_VULNDETECTOR_LOGTAG, VU_DUPLICATED_PACKAGE, agent_id, pkg_reference, cve);
+                    wm_vuldet_free_cve_node(newPkg);
+                    break;
+                case 0:
+                    mtdebug2(WM_VULNDETECTOR_LOGTAG, VU_PACKAGE_INSERT, agent_id, pkg_reference, cve);
```

Comment:
Perhaps this will flood the logs too much.

Reply:
As we discussed by chat with the team, we will keep this log during our development and testing period. After finalizing, we will decide whether to keep it.